libs/common/Memory/PoolEntry.cs

```diff
@@ -23,38 +23,27 @@ public unsafe class PoolEntry : IDisposable
         /// </summary>
         public byte* entryPtr;
 
-        readonly bool useHandlesForPin;
         GCHandle handle;
```

Comment:
Note the warnings about this not being used

Reply:
Pushed an update